src/client/sandbox/event/simulator.ts

```diff
@@ -47,6 +47,16 @@ const TOUCH_TO_POINTER_EVENT_TYPE_MAP = {
     touchmove:  'pointermove'
 };
 
+const DISABLEABLE_HTML_ELEMENT_CLASSES = [
+    HTMLButtonElement,
```

Comment:
Generally, we explicitly specify the global object when using global properties. E.g. `window.HTMLButtonElement`.

Reply:
Or even use `domUtils.isButtonElement` and so on.